lib/hunger_games/model/assigned_schedules.ex

```diff
@@ -49,7 +52,35 @@ defmodule HungerGames.AssignedSchedules do
       {:error, %Ecto.Changeset{}}
 
   """
-  def create_assigned_schedule(attrs \\ %{}) do
+  def create_assigned_schedule(attrs \\ %{})
+
+  def create_assigned_schedule(%{classes: classes} = attrs) when is_list(classes) do
+    Multi.new()
+    |> Multi.insert(
+      :insert_assigned_schedule,
+      AssignedSchedule.changeset(%AssignedSchedule{}, attrs)
+    )
+    |> Multi.insert_all(:insert_all, "assigned_schedule_classes", fn changes ->
```

Comment:
Why string and not atom?

Reply:
Because it does not work with atom since while defining a schema a string was provided